Cargo.toml

```diff
@@ -9,7 +9,7 @@ keywords = ["file-dialog", "file-picker"]
 license = "MIT"
 name = "egui_file"
 repository = "https://github.com/Barugon/egui_file"
-version = "0.8.2"
+version = "0.8.3"
```

Comment:
Since egui is a minor release with breaking changes, this also needs to be a minor release.